schema.graphql

```diff
@@ -31,6 +31,42 @@ input BelongsToFilterInput {
 
 scalar Date
 
+type Plugins implements Node {
```

Comment:
Not sure if you updated this by hand, but this was added by alex for easier reference (copypasted from the http://localhost:8080/___explore playground)

This file is not used at all in the build process, the schema is generated automatically. As far as i can tell there's no way for us to have a static schema file to be picked up by the build process.

Reply:
@aaronsteers I'd be fine with removing the file at this point. There's nothing that's in place to keep it updated anyway.

Reply:
Thanks, @rwfeather! I've removed the file per your comment.